File: app/src/main/java/com/example/fox/ratusha/data/repositories/ItemRepository.java
package com.example.fox.ratusha.data.repositories;

import com.example.fox.ratusha.data.repositories.model.ItemModele;

import java.util.List;

import io.reactivex.Flowable;

public interface ItemRepository {
    //Get all users from bd
    Flowable<List<ItemModele>> getAll();
}

File: app/src/main/java/com/example/fox/ratusha/data/repositories/ItemRepository.kt
package com.example.fox.ratusha.data.repositories

import com.example.fox.ratusha.ui.entity.ItemOrder
import com.example.fox.ratusha.ui.entity.Order
import com.example.fox.ratusha.ui.entity.TownHall
import io.reactivex.Flowable

interface ItemRepository {
    fun setForpostItem(order: List<Order>)
    fun getInfoTownHall(): Flowable<List<TownHall>>
    fun getItemForpost(): Flowable<List<ItemOrder>>
    fun getItemOctal(): Flowable<List<ItemOrder>>
}
